pkg/docker/config/config.go

```diff
@@ -288,13 +306,13 @@ func getCredentialsWithHomeDir(sys *types.SystemContext, registry, homeDir strin
 // GetCredentials API. The new API should be used and this API is kept to
 // maintain backward compatibility.
 func GetAuthentication(sys *types.SystemContext, registry string) (string, string, error) {
-	return getAuthenticationWithHomeDir(sys, registry, homedir.Get())
+	return getAuthenticationWithHomeDir(sys, nil, registry, homedir.Get())
 }
 
 // getAuthenticationWithHomeDir is an internal implementation detail of GetAuthentication,
 // it exists only to allow testing it with an artificial home directory.
-func getAuthenticationWithHomeDir(sys *types.SystemContext, registry, homeDir string) (string, string, error) {
-	auth, err := getCredentialsWithHomeDir(sys, registry, homeDir)
+func getAuthenticationWithHomeDir(sys *types.SystemContext, ref reference.Named, registry, homeDir string) (string, string, error) {
```

Comment:
I wouldn’t bother with the new parameter here , and just call `getCredentialsWithHomeDir(…, nil, …)`. This is a legacy compatibility path with no new callers expected.

Reply:
Yep, fixed as suggested :+1: